tools/quake3/common/vfs.c

```diff
@@ -289,6 +289,7 @@ int vfsLoadFile( const char *filename, void **bufferptr, int index ){
 
 		f = fopen( filename, "rb" );
 		if ( f == NULL ) {
+			fclose(f);
```

Comment:
I'm pretty sure it's undefined behaviour to call `fclose` on a null pointer. That's not to speak of the fact that it's completely pointless to attempt to close a file that was never opened.

Reply:
hmm, that's right!